src/game/clock.c

```diff
@@ -5,6 +5,26 @@
 #include <stdio.h>
 #include <time.h>
 
+#define TURBO_NONE 1
+#define TURBO_DOUBLE 2
+#define TURBO_TRIPLE 3
```

Comment:
The code never uses `TURBO_DOUBLE`. Please replace these defines with `#define MAX_TURBO_SPEED_MUL 3`.

(Alternatively we could have a `switch` statement but it seems to be an overkill.)